pkgs/applications/audio/midas/generic.nix

```diff
@@ -12,25 +12,48 @@ stdenv.mkDerivation rec {
   dontBuild = true;
   dontStrip = true;
 
+  nativeBuildInputs = [
+    copyDesktopItems
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "${pname}";
```

Comment:
I'm not sure how I feel about this. Firstly, the quotes and the braces are completely unnecessary as you can just use pname directly, but I don't think using pname is a good idea at all — theoretically someone can just override pname and ruin everything. 

Given this is a 'generic' version of a bunch of different packages, maybe you could accept differently-named parameters that are hard to accidentally override?